prow/cmd/deck/static/prow/prow.ts

```diff
@@ -397,6 +397,7 @@ function escapeRegexLiteral(s: string): string {
 }
 
 function redraw(fz: FuzzySearch): void {
+    const rerunStatus = getParameterByName("rerun");
```

Comment:
As a general architectural point, I don't think we want to implement this by having users follow links and being redirected around (and indeed the current implementation based on this scheme in #12827 is unsafe).

Assuming we ultimately end up hitting an API via XHR, does it still make sense to have this query supported and carried around?